vcx/libvcx/src/api/vcx.rs

```diff
@@ -153,6 +153,19 @@ fn _finish_init(command_handle: CommandHandle, cb: extern fn(xcommand_handle: Co
                 cb(command_handle, e.into());
             }
         }
+
+        match settings::get_config_value(settings::CONFIG_WEBHOOK_URL) {
+            Ok(webhook_url) => match ::messages::agent_utils::update_agent_webhook(&webhook_url) {
+                Ok(()) => debug!("Agent webhook url updated on init"),
```

Comment:
- Should call the callback here. 
```
cb(command_handle, error::SUCCESS.code_num);
```
Consequently you also have:
- remove callback from open_wallet success branch
- add return statement to open_wallet error branch

- Also I'd suggest to call `info!` with information about the url which has been registered.